src/PlaywrightSharp.Tests/Page/GoToTests.cs

```diff
@@ -78,7 +82,7 @@ public async Task ShouldCaptureIframeNavigationRequest()
             Assert.Equal(TestConstants.ServerUrl + "/frames/one-frame.html", response.Url);
 
             Assert.Equal(2, Page.Frames.Length);
-            Assert.Same(Page.FirstChildFrame(), requestFrame);
+            Assert.Same(Page.Frames[1], requestFrame);

```

Comment:
Why are we changing this?

Reply:
this is the correct way, how playwright is doing it

Reply:
We brought this recipe from Puppeteer. In .NET the order of the frames is not guaranteed. Browser callbacks are processed in parallel an the `ConcurrentDictionary` does not guarantee an order.